extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -93,11 +112,12 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 			// Initilaize Options Tab
 			this.optionsTab = {
 				title: localizedConstants.OptionsSectionHeader,
-				id: 'options',
+				id: this.optionsTabId,
 				content: this.createGroup('', this.optionsTabSectionsContainer, false)
 			};
 
 			// Initilaize tab group with tabbed panel
+			const disposables: vscode.Disposable[] = [];
```

Comment:
And what are you doing with this? Make sure it's being disposed once it's no longer needed (when the dialog is closed most likely)